README.md

```diff
@@ -12,11 +12,14 @@ There are two, separate active Boost projects.
 
 with a Boost Gradle project under development.
 
+### Using the Boost Maven Plugin
+Please see the [boost-maven plugin documentation](https://github.com/MicroShed/boost/tree/master/boost-maven#boost-maven-plugin-prototype) for details on using boost in your maven application project.
+
 ### Developing Boost
 
 If you are interested in contributing to Boost, read the [wiki](https://github.com/dev-tools-for-enterprise-java/boost/wiki) for more information.
```

Comment:
https://github.com/dev-tools-for-enterprise-java/boost/wiki =>
https://github.com/MicroShed/boost/wiki

(I know you didn't change this)